source/en/fv_convert.rst

```diff
@@ -575,6 +575,71 @@ It depends on "type" how to specify weight and name features.
  :type:   Specifies the name of extractor in use. The extractor is either one defined in "binary_types". Note that no pre-defined extractors are prepared.
 
 
+Feature Extraction from Combination Data
+-------------------------------
+
+We can make new combination features by combining number features oｒ string features. 
+As with strings and numbers, feature extraction rules are also described for combination types.
+We can make user-defined extractors for combination types, too.
+
+.. code-block:: js     
```

Comment:
It is better to describe the meaning of this example configuration, like in the Japanese document.